packages/evolution-backend/src/models/migrations/20230921100001_setInterviewSurvey.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { _isBlank } from 'chaire-lib-common/lib/utils/LodashExtensions';
+import { Knex } from 'knex';
+import { SurveyAttributes } from '../../services/surveys/survey';
+import config from 'chaire-lib-common/lib/config/shared/project.config';
+
+const interviewTable = 'sv_interviews';
+const surveyTable = 'sv_surveys';
+
+export async function up(knex: Knex): Promise<unknown> {
+    // Add the survey column to the table
+    await knex.schema.alterTable(interviewTable, (table: Knex.TableBuilder) => {
+        table.integer('survey_id');
+    });
+
+    // Get existing surveys:
+    const existingSurveys = await knex
+        .select('*')
+        .from(surveyTable);
+
+    const surveys : SurveyAttributes[] = [];
+    for (let i = 0; i < existingSurveys.length; i++) {
+        const survey = existingSurveys[i] as SurveyAttributes;
+        surveys.push(survey);
+    }
+
+    // this is the survey id to which we will assign all interviews:
+    let assignedSurveyId : number | undefined = undefined;
+
+    // If only one survey exist, use this one
+    if (surveys.length === 1) {
+        assignedSurveyId = surveys[0].id;
+    } else {
+        // if more then 1 survey or no survey in db, create a default one using the project shortname or set to string "default":
+        const returning = await knex(surveyTable)
```

Comment:
You should just take the first one, otherwise, you will have a new survey every time you re-run this (which should happen only while developing, but still...)

Reply:
done